third_party/2and3/markdown/core.pyi

```diff
@@ -1,14 +1,23 @@
-from typing import Any, BinaryIO, Mapping, Optional, Sequence, Text, TextIO, Union
+from typing import Any, BinaryIO, Callable, Dict, List, Mapping, Optional, Sequence, Text, TextIO, Union
 from typing_extensions import Literal
+from xml.etree.ElementTree import Element
 
+from .blockparser import BlockParser
 from .extensions import Extension
-from .util import Registry
+from .util import HtmlStash, Registry
 
 class Markdown:
     preprocessors: Registry
     inlinePatterns: Registry
     treeprocessors: Registry
     postprocessors: Registry
+    parser: BlockParser
+    htmlStash: HtmlStash
+    output_formats: Dict[Literal["xhtml", "html"], Callable[[Element], Text]]
```

Comment:
Fixed. (It may be a Python-Markdown bug that this is shared between instances, though: there’s a test that does `md.output_formats['fake'] = fakeSerializer`.)